### DISCUSSION TITLE
preserveFilenames convert image to jpg ?

### DISCUSSION THREAD

**Discussion by erenergul** [filamentphp/filament#2260] [General]
I tried to upload webp image but when i use preserveFilenames. It will convert it jpg? I dont understand why ?

**Comment by danharrin** [comment]
Could be FilePond?

**Reply by erenergul** [reply]
Im sorry Dan. I still dont understand behaviour of filepond. If I dont use preserveFilenames. It doesnt convert jpg. 

   ```
FileUpload::make('featured_image')
                           ->preserveFilenames()
```
So if i dont use preserverdFileName. It is not still FilePond ? Why it doesn't convert.

**Comment by erenergul** [comment]
I also try to add 24kb image. And it converted to me 42kb and upload it ? What is wrong with it ? I try to rendered smaller but it returned me bigger ? If it is filepond. That is really stupid things i ever seen.

**Reply by danharrin** [reply]
Can you attach a demo image in a reply so I can test it myself? For both issues.

**Reply by erenergul** [reply]
> Can you attach a demo image in a reply so I can test it myself? For both issues.

https://www.veed.io/view/2193a7c7-3518-48fd-b92c-21832f7442ae?sharingWidget=false

I just put a video about it. As you see on this video. File is 15kb but after click create button. It returned me 24kb. This is sample 

FileUpload::make('featured_image')

I didn't do anything else. Just basic file upload.,

**Reply by danharrin** [reply]
I cannot test it myself and fix without the files to test with.

**Reply by erenergul** [reply]
> I cannot test it myself and fix without the files to test with.

I didnt understand what do you want exactly from me ? You want images and I sent you video to clearify everything. 

`FileUpload::make('anyname')`

Just use this one on any system you want. You will see they change image size. I want to understand WHY ?

I upload every images manual to storage. I don't think i will use fileupload system again. If you don't fix or dont understand whats the issue. Thats fine. I will keep upload manual. 

I like filament. Its really good and easy. Maybe this is not a bug. Maybe its normal behaviour file upload system but Its wrong for me. 

Hope you understand. Thank you.

**Reply by danharrin** [reply]
I believe that #2453 might fix this? Btw, it's a Filepond and not Filament issue
